cla_backend/libs/eligibility_calculator/cfe_civil/savings.py

```diff
@@ -0,0 +1,30 @@
+
+def translate_savings(savings_data):
+    liquid_capital = []
+    non_liquid_capital = []
+    if savings_data.bank_balance > 0:
+        liquid_capital.append(
+            {
+                "value": savings_data.bank_balance / 100,
+                "description": "Savings",
+                "subject_matter_of_dispute": False
+            })
+    if savings_data.investment_balance > 0:
+        liquid_capital.append(
+            {
+                "value": savings_data.investment_balance / 100,
+                "description": "Investment",
+                "subject_matter_of_dispute": False
+            })
+    if savings_data.asset_balance > 0:
+        non_liquid_capital.append(
+            {
+                "value": savings_data.asset_balance / 100,
+                "description": "Asset",
+                "subject_matter_of_dispute": False
+            })
```

Comment:
@starswan As per the [sheet](https://docs.google.com/spreadsheets/d/12C0GN99rQjZwUsbyfuaNNzHCmIsMwVezHcV_NqDVUMg/edit#gid=0) check for `credit_balance ` is missing. What do you think for adding a  `credit_balance` check condition with comment even if its not applicable

Reply:
I can't work out what to do with it. I think the best thing we can do is raise a ticket so we don't forget it

Reply:
@starswan raised [LEP-453](https://dsdmoj.atlassian.net/browse/LEP-453) to cover credit_balance